webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests Step 7
+
+@pytest.mark.parametrize("element", ["<a href=\"/webdriver/tests/element_click/support/input.html\">123456</a>",
+                                     """<p style="background-color: rgb(255, 255, 0); width: 5em;">
+                                        <a href="/webdriver/tests/element_click/support/input.html">Helloooooooooooooooooooo Worlddddddddddddddd</a>
+                                    </p>"""])
+
+def test_click_link(session, element):
+    # 14.1 Step 7
+    session.url = inline(element)
+    element = session.find.css("a", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    element = session.find.css("input", all=False)
+    assert element.attribute("value") == "Hello World"
+
+
+def test_click_link_unload_event(session):
+    # 14.1 Step 7
+    session.url = inline("""<body onunload="checkUnload()">
+                                <a href="/webdriver/tests/element_click/support/input.html">click here</a>
+                                <input type=checkbox>
+                                <script>
+                                function checkUnload() {
+                                    document.getElementsByTagName("input")[0].checked = true;
+                                }
+                                </script>
+                            </body>""")
+    element = session.find.css("a", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    element = session.find.css("input", all=False)
+    assert element.attribute("value") == "Hello World"
+
+    session.back()
+
+    element = session.find.css("input", all=False)
+    response = session.execute_script("return document.getElementsByTagName(\"input\")[0].checked;")
+    assert response is True
+
+
+def test_click_link_hash(session):
+    # 14.1 Step 7
+    session.url = inline("<a href=\"#\">aaaa</a>")
```

Comment:
Use `"""<a href="#">aaaa</a>"""` to avoid having backslashes to escape the inner quotes.

Reply:
fixed